### PR TITLE
Mer Template

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Description

Had to update image base template due to bandbin and kernel group not existing in mer cube label. This required updating existing templates that use this base template.

This is a completed template. MER did not have any edge cases and is producing identical output cubes as mer2isis.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/appdata/import/CassiniISS.tpl [hunk 1]
```diff
@@ -172,22 +172,26 @@ ObservationId = {{ OBSERVATION_ID.Value }}
...
-OriginalBand = 1
-{% set cassiniIssBandInfo = CassiniIssBandInfo(INSTRUMENT_ID.Value, filterName.0, filterName.1) %}
-Center       = {{ cassiniIssBandInfo.0 }}
-Width        = {{ cassiniIssBandInfo.1 }}
-{% endblock%}
+{% block additional_groups %}
+Group = BandBin
+  {% if exists("FILTER_NAME") %}
+  {% set filterName=FILTER_NAME.Value %}
+  FilterName   = {{ filterName.0 }}/{{ filterName.1 }}
```
Comment (gsn9) [inline isis/appdata/import/CassiniISS.tpl:179]: should this `{{ filterName.0 }}/{{ filterName.1 }}` be this `{{ filterName.0 }}{{ filterName.1 }}`?

File: isis/appdata/import/MerMI.tpl [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+{% block instrument %}
+    {% if exists("ROVER_MOTION_COUNTER") %}
+    {% set motionCounter=ROVER_MOTION_COUNTER.Value %}
+    RoverMotionCounter          = ({% for count in motionCounter %}
+                                    {% if loop.is_last %}
+                                    {{ count }}
+                                    {% else %}
+                                    {{ count }},
+                                    {% endif %}
+                                   {% endfor %})
```
Comment (gsn9) [inline isis/appdata/import/MerMI.tpl:12]: is this if/else block needed

File: isis/appdata/import/MerMI.tpl [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    {% endif %}
+    {% if exists("ROVER_MOTION_COUNTER_NAME") %}
+    {% set counterName=ROVER_MOTION_COUNTER_NAME.Value %}
+    RoverMotionCounterName      = ({% for name in counterName %}
+                                    {% if loop.is_last %}
+                                    {{ name }}
+                                    {% else %}
+                                    {{ name }},
+                                    {% endif %}
+                                   {% endfor %})
```
Comment (gsn9) [inline isis/appdata/import/MerMI.tpl:24]: same here

File: isis/appdata/import/MerMI.tpl [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                                  NULL
+                                  {% endif %}
+    StopTime                    = {% if exists("STOP_TIME") %}
+                                  {% set stopTime=STOP_TIME.Value %}
+                                  {{ RemoveStartTimeZ(stopTime) }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    ExposureDuration            = {% if exists("INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION") %}
+                                  {{ INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION.Value }}<ms>
```
Comment (gsn9) [inline isis/appdata/import/MerMI.tpl:90]: Should there be a space between `{{ INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION.Value }}` and `<ms>`?

File: isis/appdata/import/MerMI.tpl (hunk 2 above)
Comment (amystamile-usgs) [reply isis/appdata/import/MerMI.tpl:12]: It is for determining if a comma should be added after a number. For example, the desired output is `(1, 2, 3, 4, 5)`. So the if statement is checking if it's the last element, if not add a comma.

File: isis/appdata/import/MerMI.tpl (hunk 4 above)
Comment (amystamile-usgs) [reply isis/appdata/import/MerMI.tpl:90]: Good catch. It seems like the label ingestion is still handling this correctly. But for consistency, I can add in a space.

File: isis/appdata/import/CassiniISS.tpl (hunk 1 above)
Comment (amystamile-usgs) [reply isis/appdata/import/CassiniISS.tpl:179]: The desired output is `CO-S-ISSNA/ISSWA-2-EDR-V1.0` so that is why we have a dash.

**Review by scsides — APPROVED** [review]
Fixed the same line edit conflict and will merge

**Pull request merged**